backends/calico/client.go

```diff
@@ -90,7 +90,7 @@ func NewCalicoClient(configfile string, routeReflector bool) (*client, error) {
 		cacheRevision:     1,
 		revisionsByPrefix: make(map[string]uint64),
 		nodeMeshEnabled:   nodeMeshEnabled,
-		routeReflector:    routeReflector,
+		routeReflector:    true, // CD TODO: routeReflector,
```

Comment:
That reminds about calico/routereflector's confd having to watch all of the Node resources in the cluster; whereas calico/node (until now) doesn't.  I wonder if that might be a reason not to combine calico/node and RR function as you are exploring.

Reply:
Yep, that's one of the consequences we need to explore a bit. Could have scale implications if we're not careful.